docs/sources/patterns/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+aliases:
+  - /docs/explore-apps/logs/patterns
+description: 
```

Comment:
FYI `description` is required.  This provide a short description for both Google search results and search results in our docs/website.